themes/dark_pride.toml

```diff
@@ -0,0 +1,39 @@
+# Dark Pride
+# A dark trans pride colour inspired theme
+
+# Original author: apathyrecharge
+# Ported by: Stella-Zephyr
+# Ported from: kovidgoyal / kitty-themes
+# License: GPLv3
```

Comment:
This section can just be removed. The reference to the original is available in the theme overview and licensing a colorscheme under GPLv3 is nonsense anyway.

Reply:
Alright, commited and pushed a version without that part, let me know if anything else needs updating.